### ISSUE TITLE
Automatically enable WSL on first install

### ISSUE THREAD

**Issue by rhaist** [microsoft/WSL-DistroLauncher#20]
After reading some of the comments received for the Debian store app, I think it would benefit our users greatly to not have to follow the manual enablement mechanism for fist users of wsl.

Could we integrate a check in the launcher to see if the wsl optional windows feature has been enabled already, and if not, to show a user friendly pop-up?

This would lower the barrier for non-technical users significantly.

**Comment by julianxhokaxhiu** [comment]
But still a suggestion or a link to an how-to would be highly appreciated. Or find a way to enable the flag and then ask for explicit reboot.

I second the idea to lower down the barrier. Would make WSL definitely a game changer.

**Comment by benhillis** [comment]
@julianxhokaxhiu - There is already a link that is displayed:
https://github.com/Microsoft/WSL-DistroLauncher/blob/e9a4504514223b27624eedaf4fdc8708cfdbaecd/DistroLauncher/messages.mc#L79

**Comment by levrik** [comment]
@benhillis Sure that this one works? There are many complains in the Microsoft Store about an unknown error and I also remember that I got that when I first set it up.

**Comment by benhillis** [comment]
@levrik - if it doesn't work for you let me know and I can investigate. That's how I enable the subsystem on my test machines.

**Comment by levrik** [comment]
@benhillis No. Enabling the subsystem and installing everything works perfect. But this error message doesn't seem to work if the subsystem is still disabled. It shows some cryptic error instead. At least when I set it up for the first time and there are also a few complains on the Microsoft Store about this error.

**Comment by benhillis** [comment]
@levrik - This errors on the Microsoft Store are from a previous version of this sample that did not have proper error checking.

**Comment by levrik** [comment]
@benhillis ah, okay. good to know. then there should be no issue with that message 🙂

**Comment by cpus133525** [comment]
However, according to my observation, some people just do as follows:
1. Install a Linux "app" from the Store without reading any attention message (Yes, just clicking the download button and waiting for it completed).
2. Open the Linux "app" and find the error message.
3. Uninstall the Linux "app" and rate it 1 star in the store.
4. Tell others not to use the Linux "apps" from the store because he/she thinks it buggy.
